From 94e3ee8893df65428470b5701eab4ecbddf4ffe1 Mon Sep 17 00:00:00 2001
From: Nikita Enin <n.enin@samsung.com>
Date: Mon, 13 Feb 2023 13:36:04 +0300
Subject: [PATCH 19/24] Remove default value for weights_path

Signed-off-by: Nikita Enin <n.enin@samsung.com>
---
 run_test.sh | 10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

diff --git a/run_test.sh b/run_test.sh
index 5e68d8e..e299fb9 100755
--- a/run_test.sh
+++ b/run_test.sh
@@ -14,7 +14,6 @@ num_indices_per_lookup_fixed=1
 interaction_op="cat"
 num_batches=100
 axdimm_tensor_type="simple"
-weights_path="/var/lib/jenkins/workspace/axdimm_ci/base/input/DLRM_FLOAT/embedded.bin"
 script_run_command="python3"
 script_dir=$(dirname "$0")
 
@@ -37,6 +36,15 @@ esac
     shift
 done
 
+weights_err_msg="Please provide correct weights path: --weights-path /path/to/weights/file"
+if [ -z "${weights_path}" ]; then
+    echo "--weights-path is requred. ${weights_err_msg}"
+    exit -1
+elif [ ! -f "${weights_path}" ]; then
+    echo "File '${weights_path}' doesn't exist. ${weights_err_msg}"
+    exit -1
+fi
+
 $script_run_command $script_dir/dlrm_s_pytorch.py --arch-embedding-size="$embedding_size" --arch-mlp-bot="$arch_mlp_bot" --arch-mlp-top="$arch_mlp_top" \
                             --arch-sparse-feature-size="$sparse_feature_size" --mini-batch-size="$mini_batch_size" --num-indices-per-lookup-fixed="$num_indices_per_lookup_fixed" \
                             --num-indices-per-lookup="$num_indices_per_lookup" --arch-interaction-op="$interaction_op" --num-batches="$num_batches" \
-- 
2.34.1

