From 45c008fd3e402f371ceb76ec70350d6ec0ab11ec Mon Sep 17 00:00:00 2001
From: Savelii Motov <s.motov@partner.samsung.com>
Date: Mon, 26 Sep 2022 19:06:13 +0300
Subject: [PATCH 042/225] [drivers/dax/axdimm] Add relative paths of files

Partial: AXDIMM-313

Signed-off-by: Savelii Motov <s.motov@partner.samsung.com>
---
 include/uapi/linux/libaxdimm.h | 37 ++++++++++++++++++++++++++++++++++
 1 file changed, 37 insertions(+)

diff --git a/include/uapi/linux/libaxdimm.h b/include/uapi/linux/libaxdimm.h
index a38e69215..a6a54055d 100644
--- a/include/uapi/linux/libaxdimm.h
+++ b/include/uapi/linux/libaxdimm.h
@@ -73,6 +73,43 @@
 /* Path to AXDIMM meminfo DD interface */
 #define MEMINFO_PATH "/dev/axdimm_meminfo"
 
+/* Path to dax sysfs root */
+#define DAX_SYSFS_ROOT "/sys/bus/dax/drivers/device_dax"
+
+// Block of sysfs relative paths for device
+// DAX_SYSFS_ROOT"/dax%d.%d/ranks"
+
+/* O_WRONLY Path to reset device, write "1" for reset */
+#define DEVICE_RESET_PATH "reset"
+/* O_RDWR Path to acquisition timeout, measured in ns */
+#define DEVICE_ACQUISITION_TIMEOUT_PATH "acq_timeout"
+
+// Block of sysfs paths for ranks info, all O_RDONLY
+// DAX_SYSFS_ROOT"/dax%d.%d/ranks/%d
+
+/* Rank state, 0 = free, 1 = busy */
+#define RANK_STATE_PATH "state"
+/* Rank acquisitions count */
+#define RANK_ACQUISITION_COUNT_PATH "acquisition_count"
+/* Get free size in bytes */
+#define RANK_FREE_SIZE_PATH "free_size"
+
+#define RANK_REGION_SIZE_PATH(region) "regions/" #region "/size"
+#define RANK_REGION_OFFSET_PATH(region) "regions/" #region "/offset"
+
+/* BASE region's size and offset */
+#define RANK_REGION_BASE_SIZE_PATH RANK_REGION_SIZE_PATH(base)
+#define RANK_REGION_BASE_OFFSET_PATH RANK_REGION_OFFSET_PATH(base)
+/* INST region's size and offset */
+#define RANK_REGION_INST_SIZE_PATH RANK_REGION_SIZE_PATH(instr)
+#define RANK_REGION_INST_OFFSET_PATH RANK_REGION_OFFSET_PATH(instr)
+/* CFGR region's size and offset */
+#define RANK_REGION_CFGR_SIZE_PATH RANK_REGION_SIZE_PATH(cfgr)
+#define RANK_REGION_CFGR_OFFSET_PATH RANK_REGION_OFFSET_PATH(cfgr)
+/* PSUM region's size and offset */
+#define RANK_REGION_PSUM_SIZE_PATH RANK_REGION_SIZE_PATH(psum)
+#define RANK_REGION_PSUM_OFFSET_PATH RANK_REGION_OFFSET_PATH(psum)
+
 /* The enumeration of axdimm blocks addresses */
 typedef enum _axd_mem_blocks_e {
 	AXDIMM_BLOCK_BASE = 0,
-- 
2.34.1

