From 0d52ecd903026d06ed67d5c6e8ad9948d09062e9 Mon Sep 17 00:00:00 2001
From: Savelii Motov <s.motov@samsung.com>
Date: Tue, 29 Nov 2022 16:03:20 +0300
Subject: [PATCH 069/161] [drivers/dax/axdimm] fix -Wstrict-prototype error for
 clang

---
 drivers/dax/axdimm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/dax/axdimm.c b/drivers/dax/axdimm.c
index 266b27122..2e5ceab19 100644
--- a/drivers/dax/axdimm.c
+++ b/drivers/dax/axdimm.c
@@ -404,7 +404,7 @@ int init_axdimm_device(struct dev_dax *dev_dax)
 				     &axdimm_proc_mgr);
 }
 
-void cleanup_axdimm_device()
+void cleanup_axdimm_device(void)
 {
 	AXDIMM_DBG("Cleaning up AXDIMM device\n");
 
-- 
2.34.1

