From 8b17ed015e937522e2501d9257511425703f5b72 Mon Sep 17 00:00:00 2001
From: Savelii Motov <s.motov@samsung.com>
Date: Thu, 15 Dec 2022 10:58:46 +0300
Subject: [PATCH 073/225] [config-axdimm] fix: enable caching for 9P_fs

Signed-off-by: Savelii Motov <s.motov@samsung.com>
---
 config-axdimm | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/config-axdimm b/config-axdimm
index 265373515..4724582d8 100644
--- a/config-axdimm
+++ b/config-axdimm
@@ -10165,7 +10165,7 @@ CONFIG_OVERLAY_FS_XINO_AUTO=y
 #
 CONFIG_NETFS_SUPPORT=y
 CONFIG_NETFS_STATS=y
-CONFIG_FSCACHE=m
+CONFIG_FSCACHE=y
 CONFIG_FSCACHE_STATS=y
 # CONFIG_FSCACHE_DEBUG is not set
 CONFIG_CACHEFILES=m
@@ -10395,6 +10395,7 @@ CONFIG_AFS_FS=m
 CONFIG_AFS_FSCACHE=y
 # CONFIG_AFS_DEBUG_CURSOR is not set
 CONFIG_9P_FS=y
+CONFIG_9P_FSCACHE=y
 CONFIG_9P_FS_POSIX_ACL=y
 CONFIG_9P_FS_SECURITY=y
 CONFIG_NLS=y
-- 
2.34.1

