From a5b73d4fe2d0ed95fa8631d1e3d405159a95cfde Mon Sep 17 00:00:00 2001
From: Savelii Motov <s.motov@samsung.com>
Date: Fri, 7 Apr 2023 18:46:48 +0300
Subject: [PATCH 097/161] [dax] udpdate install_dax_drivers script

Signed-off-by: Savelii Motov <s.motov@samsung.com>
---
 install_dax_drivers.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/install_dax_drivers.sh b/install_dax_drivers.sh
index 4dbb77474..ce3ab7c98 100755
--- a/install_dax_drivers.sh
+++ b/install_dax_drivers.sh
@@ -7,5 +7,5 @@ sudo cp -v drivers/dax/hmem/*.ko /lib/modules/`uname -r`/kernel/drivers/dax/hmem
 sudo cp -v include/uapi/linux/libaxdimm.h /usr/include/linux
 sudo cp -v include/uapi/linux/libaxdimm.h /usr/src/linux-headers-`uname -r`/include/uapi/linux/libaxdimm.h
 
-sudo rmmod device_dax
-sudo modprobe device_dax "$@"
+lsmod | grep device_dax > /dev/null && sudo rmmod device_dax
+sudo modprobe -v device_dax "$@"
-- 
2.34.1

